ui/src/pages/browse.tsx

```diff
@@ -63,19 +64,31 @@ const Browse: Types.NextPage<Props> = (props): React.ReactElement => (
             <section id="content" className="container mx-auto grid grid-cols-1 px-8 lg:grid-cols-8 lg:gap-16">
                 <aside className="relative col-span-2 hidden lg:block">
                     <div className="sticky top-16">
-                        <h2 className="mb-6 block font-montserrat text-xl font-bold leading-none text-grey-800 transition-colors duration-500 dark:text-white-50">
-                            Publication type
-                        </h2>
-                        <Components.Link
+                        {/* view all publication & authors buttons */}
+                        <Components.Button
+                            link
                             href={`${
                                 Config.urls.search.path
                             }?for=publications&type=${Config.values.publicationTypes.join()}`}
-                            className="group mb-2 block w-fit rounded border-transparent outline-0 focus:ring-2 focus:ring-yellow-400"
-                        >
-                            <span className="font-medium text-grey-800 transition-colors duration-500 group-hover:text-grey-500 dark:text-grey-50">
-                                All
-                            </span>
-                        </Components.Link>
+                            title="View all publications"
+                            iconPosition="RIGHT"
+                            icon={
+                                <OutlineIcons.ArrowRightIcon className="h-4 w-4 text-teal-500 transition-colors duration-500 dark:text-white-50" />
+                            }
+                        />
+                        <Components.Button
+                            link
+                            href={`${Config.urls.search.path}?for=users`}
+                            title="View all authors"
+                            iconPosition="RIGHT"
+                            icon={
+                                <OutlineIcons.UserIcon className="h-4 w-4 text-teal-500 transition-colors duration-500 dark:text-white-50" />
+                            }
+                            className="mb-6"
+                        />
```

Comment:
This seems to have had a side effect and shuffled everything on to one line.

![Screenshot 2022-05-19 at 16 29 58](https://user-images.githubusercontent.com/86601264/169337403-1fe02421-2e00-43a1-afde-c71713867df2.png)